### PR TITLE
[2.19.x] DDF-6037 G-1625 Fix CONTAINS Filter Behavior with Multi-word Filters

### PR DESCRIPTION
**Pull request by bennuttle**
#### What does this PR do?
Addresses https://github.com/codice/ddf/issues/6037 by appending wildcards to the filter to properly handle CONTAINS operations and changes the matching to consider the entire string rather than addressing each token individually.

#### Who is reviewing it? 

@leo-sakh 
@lavoywj 
@frnkshin 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining

#### How should this be tested?

See steps to reproduce in https://github.com/codice/ddf/issues/6037

#### What are the relevant tickets?
Fixes: #6037

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/js/filter.js [hunk 1]
```diff
@@ -64,26 +64,19 @@ function checkToken(token, filter) {
 }
 
 function matchesILIKE(value, filter) {
-  const valueToCheckFor = filter.value.toLowerCase()
+  filter.value = filter.value.toLowerCase()
```
Comment (millerw8) [inline ui/packages/catalog-ui-search/src/main/webapp/js/filter.js:67]: maybe an naive question, but I am going to ask anyway.... by modifying the filter.value, do we end up changing the user's input they made?  I am hoping the answer is no, but I have seen it leak before (like in default search titles)

Comment (bennuttle) [reply ui/packages/catalog-ui-search/src/main/webapp/js/filter.js:67]: Double-checked this and the value is not leaking back to the UI or changing the user's entered input

**Comment by bennuttle** [comment]
build now

**Comment by frnkshin** [comment]
I heroed this build.

Steps taken:
1. Ingest sample data into DDF build from @bennuttle 's branch.
2. Run search `anytext CONTAINS The United States of America`
3. Confirmed that results show with the query
4. Apply filter `anytext CONTAINS United States`
5. Confirmed that front end filter works by showing results with text `United States`

**Pull request merged**